packages/section/src/section.tsx

```diff
@@ -0,0 +1,39 @@
+import * as React from 'react';
+
+type DivElement = React.ElementRef<'table'>;
```

Comment:
This should be `SectionElement`

```suggestion
type SectionElement = React.ElementRef<'table'>;
```